Tests/test_file_webp_alpha.py

```diff
@@ -119,7 +119,7 @@ def test_write_unsupported_mode_PA(self):
         image.getdata()
         target = Image.open(file_path).convert("RGBA")
 
-        self.assert_image_similar(image, target, 10.0)
+        self.assert_image_similar(image, target, 25.0)
```

Comment:
That's a pretty big jump for a .0.1 release, are we sure that we're not missing something there?

Reply:
https://github.com/webmproject/libwebp/blob/master/NEWS
> - 11/24/2017: version 0.6.1
>   This is a binary compatible release.
>   * lossless performance and compression improvements + a new 'cruncher' mode
>     (-m 6 -q 100)
>   * ARM performance improvements with clang (15-20% w/ndk r15c, issue #339)
>   * webp-js: emscripten/webassembly based javascript decoder
>   * miscellaneous bug & build fixes (issue #329, #332, #343, #353, #360, #361,
>     #363)
>   Tool updates / additions:
>     added webpinfo - prints file format information (issue #330)
>     gif2webp - loop behavior modified to match Chrome M63+ (crbug.com/649264);
>                '-loop_compatibility' can be used for the old behavior

Here's a copy of the generated file from my local machine using 0.6.1.
![transparent](https://user-images.githubusercontent.com/3112309/33787157-1d5041ac-dcc0-11e7-9b49-db905aca4c39.png)

Reply:
I have no idea where those issues actually reside on the web, other than not on github, and not in the repo it's cloned from. 

Anyway, it looks like there was a YUV->RGB change in 0.52 that could potentially account for that.